### DISCUSSION TITLE
Allow users to switch between light and dark variants of a template

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [NamelessMC/Nameless#2228] [Ideas]
This will be I think so easy to implement:
so each user can choose between themes that are enabled by admins, like on XenForo(2), so people can choose between white and black (day & night mode alike).
I think this will bring a lot more to design of the site, and flexibility, also user's current theme will be cached and not forgotten so next time they access the site (for example after 1-2 days) it'll still be the same theme they have choosen.

**Comment by CqldFire** [comment]
Currently, this is available in Skyfall, but would be extremely easy to implement with either PHP or js for default revamp.

PHP would be better as it would eliminate any flickering of colors going on during page load.

I think this is a great idea, as long as PHP is used to do it :D

**Comment by samerton** [comment]
I don't imagine we'd implement anything within Nameless for templates to use - this would be controlled entirely by the template with cookies for example

I can add something to Default Revamp to allow per-user theme switching

**Comment by MotikaCraft-TheJoshua** [comment]
*I hope this is not advertising...*

![image](https://user-images.githubusercontent.com/56804505/114083168-f2319800-98ae-11eb-9b2b-27485ef1adcd.png)

![image](https://user-images.githubusercontent.com/56804505/114083215-fe1d5a00-98ae-11eb-9dda-1f845f34b445.png)

So, we all know about this in XenForo. I am sure that A LOT of us want something like this; simply because of day/night modes.
And one more thing; not everyone want to use dark mode, where some sites have "dark mode" site by default, then user want to change that to be white - he/she can't, because there is no such feature (and I think it's very simple yet very practical).
